src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -66,11 +93,14 @@
      [:div {:style {:display "block"}}
       (str "\u00A9 " yeartext " Broad Institute")
       spacer
-      [Link {:href "#" :text "Privacy Policy" :target "_self"}]
+      [Link {:href "#policy" :text "Privacy Policy" :target "_self"}]
```

Comment:
Let's remove the :target "_self" here since that is the default (and we don't do this anywhere else).